Source/WebKit/Shared/Cocoa/DefaultWebBrowserChecks.mm

```diff
@@ -71,7 +71,8 @@ static bool isInWebKitChildProcess()
         NSString *bundleIdentifier = [[NSBundle mainBundle] bundleIdentifier];
         isInSubProcess = [bundleIdentifier hasPrefix:@"com.apple.WebKit.WebContent"]
             || [bundleIdentifier hasPrefix:@"com.apple.WebKit.Networking"]
-            || [bundleIdentifier hasPrefix:@"com.apple.WebKit.GPU"];
+            || [bundleIdentifier hasPrefix:@"com.apple.WebKit.GPU"]
+            || [bundleIdentifier hasPrefix:@"com.apple.WebKit.Model"];
```

Comment:
We don't want to put the `ENABLE` guards around these things?

Reply:
Hmm, I followed places that used ENABLE(GPU_PROCESS), not sure why they (may well have been me...) didn't here, we certainly can!